<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>611</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="HLayout_show">
    <item>
     <widget class="QTextBrowser" name="textBrowser_code"/>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser_result"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_code">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>代码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_result">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>运行结果</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tree">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>语句与语法树</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_input">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>550</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>命令输入窗口</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_input">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>570</y>
     <width>611</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>614</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="HLayout_button">
    <item>
     <widget class="QPushButton" name="Button_load">
      <property name="text">
       <string>载入代码(LOAD)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button_run">
      <property name="text">
       <string>执行代码(RUN)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button_debug">
      <property name="text">
       <string>调试/单步(DEBUG/STEP)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button_clear">
      <property name="text">
       <string>清空代码(CLEAR)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>611</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTextBrowser" name="textBrowser_tree"/>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser_var"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_var">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>当前变量</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
